src/Symfony/Component/HttpKernel/Profiler/MemcacheProfilerStorage.php

```diff
@@ -11,97 +11,13 @@
 
 namespace Symfony\Component\HttpKernel\Profiler;
 
+use Symfony\Component\Profiler\Storage\MemcacheProfilerStorage as BaseMemcacheProfilerStorage;
```

Comment:
In this case i think that you can omit this use statement because `BaseMemcacheProfilerStorage` already extends `BaseMemcacheProfilerStorage` from the new component